### ISSUE TITLE
Extension author nick is hard to see

### ISSUE THREAD

**Issue by VitorLuizC** [ayu-theme/vscode-ayu#29]
The worst of them is the Ayu Dark, but the other themes is also hard to see.

**Screenshots:**
- Ayu Dark
  <img width="414" alt="captura de tela 2017-09-12 as 21 21 00" src="https://user-images.githubusercontent.com/9027363/30354070-c299296c-9800-11e7-8cbe-0c0e7ee17d62.png">

- Ayu Light
  <img width="354" alt="captura de tela 2017-09-12 as 21 25 32" src="https://user-images.githubusercontent.com/9027363/30354142-1e02fec2-9801-11e7-8c78-eafa92566bb3.png">

- Ayu Mirage
  <img width="393" alt="captura de tela 2017-09-12 as 21 25 12" src="https://user-images.githubusercontent.com/9027363/30354159-34c09980-9801-11e7-831f-42a90f3d2786.png">

**Comment by dempfi** [comment]
Yeah, I want to fix every time I get to touch the theme and every time there's no option for that. I'll double check the next time.
